docs/index.rst

```diff
@@ -2,19 +2,47 @@
 ops library API reference
 =========================
 
+The `ops` library is a Python framework for writing and testing Juju charms.
+
+  See more: `Charm SDK documentation <https://juju.is/docs/sdk>`_
+
+The library provides:
+
+- :ref:`ops_main_entry_point`, used to initialise and run your charm;
+- :ref:`ops_module`, the API to respond to Juju events and manage the application;
+- :ref:`ops_pebble_module`, the low-level API for Pebble in Kubernetes containers;
+- :ref:`ops_testing_module`, the framework for unit testing charms in a simulated environment;
+
+You can write a charm in any way you want, but with the `ops` library you get a
+framework that helps you write consistent readable charms following the latest
```

Comment:
```suggestion
framework that helps you write consistent, readable, charms following the latest
```

Reply:
+1 on that. The final comma was habit (Oxford comma all the way) but it really should be used when the list has three or more items, which isn't the case here. A list of only two items is a bit odd and indeed your suggestion of just "and" seems nicer.

Reply:
added and